internal/core/cloud/aws.go

```diff
@@ -12,12 +12,22 @@ import (
 	"fmt"
 	"io"
 	"net/http"
+	"strconv"
 	"strings"
+
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
 )
 
 const (
-	awsMetadataURL      = "http://169.254.169.254/latest/"
-	awsMetadataResource = "meta-data"
+	awsMetadataURL                = "http://169.254.169.254/latest/"
+	awsMetadataResource           = "meta-data"
+	metadataTokenTTLHeader string = "X-aws-ec2-metadata-token-ttl-seconds" //nolint
```

Comment:
question: why do we need these `nolint`?

Reply:
golang ci lint complains 🙈 

```
internal/core/cloud/aws.go:25:2: G101: Potential hardcoded credentials (gosec)
        metadataTokenTTLHeader string = "X-aws-ec2-metadata-token-ttl-seconds"
        ^
internal/core/cloud/aws.go:26:2: G101: Potential hardcoded credentials (gosec)
        metadataTokenHeader    string = "X-aws-ec2-metadata-token"
```